### PR TITLE
fix(database-change): condition search invalid

### PR DESCRIPTION
**Pull request by kiko-art**
#### What type of PR is this?
type-bug

#### What this PR does / why we need it:
fix condition search invalid
#### Which issue(s) this PR fixes:

Fixes #

#### Additional documentation e.g., usage docs, etc.:

```docs

```

### CODE REVIEW

File: server/odc-service/src/main/java/com/oceanbase/odc/service/schedule/model/QueryScheduleTaskParams.java [hunk 1]
```diff
@@ -32,7 +32,7 @@ public class QueryScheduleTaskParams {
     private List<Schedule> schedules;
     private String scheduleName;
     private Set<Long> dataSourceIds;
-    private List<TaskStatus> statuses;
+    private TaskStatus status;
```
Comment (yizhouxw) [inline server/odc-service/src/main/java/com/oceanbase/odc/service/schedule/model/QueryScheduleTaskParams.java:35]: status in can cover status equals scenario, may not neccessary to change it ?

Comment (kiko-art) [reply server/odc-service/src/main/java/com/oceanbase/odc/service/schedule/model/QueryScheduleTaskParams.java]: Changed back

**Review by yizhouxw — APPROVED** [review]
lgtm

**Review by yhilmare — APPROVED** [review]
lgtm

**Pull request merged**